NOTICE.txt

```diff
@@ -0,0 +1,7 @@
+This file list any third-party libraries or other resources that may be
+distributed under licenses different than the Azure SDK for JS software.
+
+In the event that we accidentally failed to list a required notice, please
+bring it to our attention by opening an issue.
+
+The attached notices are provided for information only.
```

Comment:
this file seems to deviate from the notice template you linked me to, is that intentional?

Reply:
@xirzec, I've updated NOTICE file to add right header. Sorry for getting back so late.